RELEASE.md

```diff
@@ -31,8 +31,10 @@ Maintaining the release branches for older minor releases happens on a best effo
   * kube-state-metrics image tag used in Kubernetes deployment yaml config.
 * Cut the new release branch, i.e., `release-1.2`, or merge/cherry-pick changes onto the minor release branch you intend to tag the release on
 * Cut the new release tag, i.e., `v1.2.0-rc.0`
+* Create a new **pre-release** on github
```

Comment:
Nice yes! Lets mention to finish release as well at the end. 
- Mark your release as released.

Reply:
Thanks, done!